src/api/services/CreateKas.ts

```diff
@@ -0,0 +1,64 @@
+import { APIResponse } from "@types";
+import API from "..";
+
+export type GetResponse = {
+  submission_id: string;
+  user: UserType;
+  payed_amount: number;
+  status: {
+    ID: number;
+    Name: string;
+  };
+  note: string;
+  evidence: string;
+  submitted_at: string;
+  updated_at: string;
+};
+
+export type UserType = {
+  npm: string;
+  name: string;
+  email: string;
+  kas_payed: number;
+};
+
+export type SubmissionRequest = {
+  user: {
+    npm: string;
+  };
+  payed_amount: number;
+  note: string;
+  evidence: string;
+};
+
+export default class CreateKasService {
+  kasPath: string = "/kas-submissions";
+  userPath: string = "/users";
+  private api: API = new API();
+  private apiForm: API = new API({ isForm: true });
+  async post(
+    submission: SubmissionRequest | string,
+  ): Promise<APIResponse<GetResponse>> {
+    const targetPath = `${this.kasPath}`;
+    try {
+      const res: APIResponse<GetResponse> = await this.apiForm.POST(
+        targetPath,
+        submission,
+      );
+      return res;
+    } catch (error) {
+      console.error("Error creating submission:", error);
+      throw error;
+    }
+  }
+  async get(queryParams: string = ""): Promise<APIResponse<UserType[]>> {
```

Comment:
```suggestion
  async get(queryParams: string): Promise<APIResponse<UserType[]>> {
```
harusnya engga perlu ini